### DISCUSSION TITLE
Brothers come to each other star repo

### DISCUSSION THREAD

**Discussion by callqh** [Schweinepriester/github-profile-achievements#430] [Q&A] [answered]
For the stars!
https://github.com/liuqh0609/note

**Comment by HHNebula** [comment, marked as answer]
Done! Do it for me, too
https://github.com/HHNebula/HHNebula

**Reply by callqh** [reply]
Done！

**Reply by callqh** [reply]
Looks like I got the Galaxy Brain for you

**Reply by callqh** [reply]
Can you create a discussion and mark my reply as answer？

**Reply by HHNebula** [reply]
Yep

**Reply by HHNebula** [reply]
https://github.com/Schweinepriester/github-profile-achievements/discussions/433
@liuqh0609
